### ISSUE TITLE
Accessibility - WCAG 2.4.6 Headings and labels, some labels are missing

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#139]
Descriptive labels should be present in forms. Also in component pages, some labels are missing.

Original version was ok.
